pkg/controller/install/install_controller.go

```diff
@@ -170,6 +149,12 @@ func (r *ReconcileInstall) install(instance *servingv1alpha1.Install) error {
 	if err := r.config.Filter(filters...).ApplyAll(); err != nil {
 		return err
 	}
+
+	// set any additional values after yaml has been applied
+	if err := r.postApply(); err != nil {
```

Comment:
So this is the right idea, but instead of *inside* the `install`, it should go *after* the call to `install` up in the `Reconcile` function. Because if `install` fails, it's going to return immediately, and nothing after it will get called anyway. That way, we only need to look at `Reconcile` to see what the operator is doing. So I was thinking instead of `postApply` here, we put something like this up in `Reconcile`...
```
if err := r.checkForMinikube(); err != nil {
  return reconcile.Result{}, err
}
if err := r.updateServiceNetwork(); err != nil {
  return reconcile.Result{}, err
}
if err := r.updateDomain(); err != nil {
  return reconcile.Result{}, err
}
```
Does that make sense? I think it'll be clearer to the folks maintaining the operator. Thoughts?

Reply:
postApply only fails if it finds something to set but it fails to set in configmap.  I thought we might want to fail the install at the point.  Or, should we still update the "Instance" Status (if we pull it out of "install")?

Reply:
I don't think we want to fail the install for that. I would expect we'd want to retry indefinitely until we can set the configmap. Unless there's some reason it'll never work, e.g. the config map isn't there.